app/src/main/java/com/geekskool/dhobi/Contracts/Contract.java

```diff
@@ -0,0 +1,99 @@
+package com.geekskool.dhobi.Contracts;
+
+import android.content.ContentUris;
+import android.net.Uri;
+import android.provider.BaseColumns;
+
+/**
+ * Created by manisharana on 17/2/17.
+ */
+public class Contract {
```

Comment:
Parking the migration tool for now. Will research on few migration tools over the weekend.